lib/sidekiq/disposal/client.rb

```diff
@@ -72,6 +72,30 @@ def drop_target?(job)
         job_in_target_set?(job, disposal_target_set(:drop))
       end
```

Comment:
Do we need these any more?

Reply:
I thought about removing it but it seems like a nice API if you wanted to build a UI for this 🤷‍♀️

Reply:
Fair point! And I DO!